Source/WTF/wtf/cocoa/RuntimeApplicationChecksCocoa.cpp

```diff
@@ -172,6 +172,14 @@ static SDKAlignedBehaviors computeSDKAlignedBehaviors()
     if (linkedBefore(dyld_spring_2022_os_versions, DYLD_IOS_VERSION_15_4, DYLD_MACOSX_VERSION_12_3))
         disableBehavior(SDKAlignedBehavior::AuthorizationHeaderOnSameOriginRedirects);
 
+    if (linkedBefore(dyld_fall_2022_os_versions, DYLD_IOS_VERSION_16_0, DYLD_MACOSX_VERSION_13_0)) {
```

Comment:
i feel like this might be better as a separate change 😅

Reply:
I can split it out, sure.